src/calculateRentalCost.js

```diff
@@ -4,7 +4,29 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalAmount = 0;
```

Comment:
You can initialise this variable with calculation like in lline 27, then make if section shorter, just return `tot-amount - discount`.